### ISSUE TITLE
Make interactive table cells focusable for accessibility

### ISSUE THREAD

**Issue by gryan337** [gryan337/zabbix-widgets-table#51]
Table cells are interactive and should probably be focusable via keyboard

**Comment by gryan337** [comment]
I think we still might need to consider adding more things to the region announcement function (i.e. cell selections, sort selections, filter applied indicator, etc?).

I also learned that certain CSS effects can cause issues for some users. I've already thought of myself as respectful and humble in many ways, but reading the WCAG guidebook is quite eye-opening. Regardless, I hope some of these changes provide additional accessibility to more users. If there is anything I am missing, please let me know!
